back/app/models/initiative.rb

```diff
@@ -58,8 +61,8 @@ class Initiative < ApplicationRecord
   with_options unless: :draft? do |post|
     post.validates :title_multiloc, presence: true, multiloc: { presence: true }
     post.validates :body_multiloc, presence: true, multiloc: { presence: true, html: true }
-    post.validates :author, presence: true, on: :publication
-    post.validates :author, presence: true, if: :author_id_changed?
+    post.validates :author, presence: true, on: :publication, unless: :anonymous?
```

Comment:
This doesn't work together with line 61

Reply:
Added some tests to check the difference between validation when draft or published